### DISCUSSION TITLE
Can github import code using a manifest file??

### DISCUSSION THREAD

**Discussion by AnishAnil** [community/community#50208] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello Team,

We are evaluating github to import code from the AOSP repositories.
I would like to know, if we can import the AOSP or other code using the manifest file into github.

This is similar to GitLab which exclusively states that manifest file import is supported in GITLAB
https://docs.gitlab.com/ee/user/project/import/manifest.html

Thank you

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @AnishAnil! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
